apps/client/src/initialize-auth-dependents.tsx

```diff
@@ -0,0 +1,18 @@
+import { v4 as uuid } from 'uuid';
+import { utcDateString } from './helpers/dates';
+import { cloudWatchMetricsRecorder } from './metrics/cloud-watch-metrics-recorder';
+import { cloudWatchLogger } from './logging/cloud-watch-logger';
+
+/**
+ * This function initializes the services that depends on authentication to work
+ * @param applicationName the name of this application
+ */
+export async function initializeAuthDependents(applicationName: string) {
+  const logStreamName = `${utcDateString(new Date())}-${uuid()}`;
+  cloudWatchMetricsRecorder.init(applicationName);
+  try {
+    await cloudWatchLogger.init(applicationName, logStreamName);
+  } catch (e) {
+    // NOOP; application should proceed;
```

Comment:
i think its worth adding a log statement here. Also the eslint rules might have an issue, the 'e' variable is not used here and build hasnt thrown error

Reply:
it default to ignore the unused error under catch https://eslint.org/docs/latest/rules/no-unused-vars#caughterrors
I think it is fine since it is a common pattern to define an `e`; enforcing this doesn't help us code faster/better